tiny-postgresql-event-store/src/main/kotlin/jp/veka/tables/Tables.kt

```diff
@@ -0,0 +1,101 @@
+package jp.veka.tables
+
+import ru.quipy.saga.SagaContext
+class EventRecordTable {
+    companion object {
+        const val name = "event_record"
```

Comment:
In you implementation we have only one table for records, in reality we want to get table per aggregate. See here the place where you ignore this fact:
```QueryBuilder.insert(eventStoreSchema, EventRecordTable.name, EventRecordDto(eventRecord, aggregateTableName))```